src/antsibull_changelog/changes.py

```diff
@@ -156,7 +156,13 @@ def save(self) -> None:
         """
         self.sort()
         self.data["ancestor"] = self.ancestor
-        store_yaml(self.path, self.data, self.config.changelog_nice_yaml)
+        sort_keys = self.config.changelog_sort == "alphanumerical"
+        store_yaml(
+            self.path,
+            self.data,
+            self.config.changelog_nice_yaml,
+            sort_keys,
```

Comment:
This would then become:
```suggestion
            nice=self.config.changelog_nice_yaml,
            sort_keys=sort_keys,
```